### DISCUSSION TITLE
Ant-Media-Server save recording in cloud

### DISCUSSION THREAD

**Discussion by htaufik** [ant-media/Ant-Media-Server#3747] [Q&A]
Hello, I want to ask something. Can Ant-media save the stream recording to the cloud like google drive? 
what I know is ant-media save the recording in usr/local/antmedia/... right?

**Comment by Mohit-3196** [comment]
Hi @htaufik,
It seems it's been rather long since this question was asked but let me write an answer just in case someone lands on this discussion :)

Yes, Ant Media Server can upload and save the recordings to all the S3 compatible buckets like AWS S3, Google Cloud Storage, Digital Ocean space, OVH object storage, Wasabi, Min.io, etc.
You can refer to this document for detailed instructions https://antmedia.io/docs/guides/recording-live-streams/s3-integration-http-forwarding/

I hope it helps.

Thank you.
